internal/server/users.go

```diff
@@ -63,6 +64,27 @@ func (a *API) GetUser(c *gin.Context, r *api.GetUserRequest) (*api.User, error)
 		return nil, err
 	}
 
+	if r.Sync {
```

Comment:
I wonder if we'd _always_ want to do this if the access key is associated to a provider. We could cache the result to avoid long requests?

Reply:
Im gonna take this approach

Reply:
Made this change